modules/formulize/class/screen.php

```diff
@@ -47,6 +49,47 @@ function __construct() {
 		$this->initVar('useToken', XOBJ_DTYPE_INT);
 	}
 
+  /**
+   * Checks if a given template name exists for the current screen.
+   * @return true if the given template name exists for the screen, false otherwise.
+   */
+  function hasTemplate($templateName) {
+      return (trim($this->getTemplate($templateName)) != "");
+  }
+
+  function getTemplate($templateName) {

```

Comment:
getTemplate already exists in this file. The current version should supersede this one, so this one can be removed.

Reply:
removed